tests/versioned/apollo-server-fastify/segments.test.js

```diff
@@ -41,16 +46,22 @@ function createFastifySegmentsTests(t, frameworkName) {
           name: `${TRANSACTION_PREFIX}//${operationPart}`,
           children: [
             {
-              name: `${OPERATION_PREFIX}/${operationPart}`,
+              name: GQL_MIDDLEWARE_SEGMENT_PATTERN,
```

Comment:
Thought about adding a helper so wouldn't need to insert this on each item but decided against it.

Seemed like it would obscure understanding of the structure too much. Also, this captured structure isn't likely to change often.